docker/dokku/Dockerfile

```diff
@@ -1,39 +1,29 @@
-FROM python:alpine3.6
+FROM python:3.7-slim
 ARG DJANGO_SETTINGS_MODULE
 ARG DJANGO_ADMIN_URL
 ARG DJANGO_SECRET_KEY
 ARG DATABASE_URL
 ARG DJANGO_SENTRY_DSN
-ENV PYTHONBUFFERED=1 DJANGO_SETTINGS_MODULE=${DJANGO_SETTINGS_MODULE} DJANGO_ADMIN_URL=${DJANGO_ADMIN_URL} DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY} DATABASE_URL=${DATABASE_URL} DJANGO_SENTRY_DSN=${DJANGO_SENTRY_DSN}
+ENV PYTHONBUFFERED=1 DJANGO_SETTINGS_MODULE=${DJANGO_SETTINGS_MODULE} DJANGO_ADMIN_URL=${DJANGO_ADMIN_URL} DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY} DATABASE_URL=${DATABASE_URL} DJANGO_SENTRY_DSN=${DJANGO_SENTRY_DSN} POETRY_VERSION=1.0.5 POETRY_VIRTUALENVS_CREATE=false
 
 # Add new user to run the whole thing as non-root
-RUN addgroup -S app \
-    && adduser -G app -h /app -D app
+RUN groupadd app
+RUN adduser --ingroup app --home /app --disabled-password --gecos "" app
+RUN adduser app app
```

Comment:
Why do we need to call `adduser` twice?

Reply:
This might actually be a relict from trying out how zo make it work with the slim build. I'm looking into this again and see if it's necessary.

Reply:
Checked it, not necessary. Amended the commit.